### ISSUE TITLE
Discrepancies between discoal paper and selection appendix

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/msprime-1.0-paper#170]
There appear to be some discrepancies between the equations we're giving in the selection appendix to the discoal paper. Specifically, the denominator of mu(p) definition differs.

@andrewkern, can you comment please?

If the model is identical to what's given in the discoal paper, is there any point in reproducing it here? It's not like we refer to the equations or anything. If not, then I'm not sure there's a lot of value in the selection appendix, as the non-equation bits could easily be merged into the main-text section.

**Comment by andrewkern** [comment]
the denominator here was rewritten to make it compatible with the code -- the way it's written in the original discoal paper, the time direction is reversed. @petrelharp and I thought it was important to put this level of detail in the appendix

**Comment by jeromekelleher** [comment]
Aha. Two questions then

- Could we merge this into the main text? There's not *that* much more detail in the appendix, and there's some overlap. 
- Should the value in the demoninator be \tanh (...) or \tan h (...)?

**Comment by andrewkern** [comment]
> * Could we merge this into the main text? There's not _that_ much more detail in the appendix, and there's some overlap.

sure!

> * Should the value in the demoninator be \tanh (...) or \tan h (...)?

\tanh

**Comment by andrewkern** [comment]
e.g. the hyperbolic tangent function

**Comment by jeromekelleher** [comment]
Great, thanks. Will do after #169 is merged
